hotel-management/src/components/Dropdown/DropdownFilterMenu.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+
+import { DropdownItem } from '@/types/dropdown';
+
+import { Button } from '../Button';
+import './index.css';
+
+type DropdownProps = {
+  label: string;
+  items: DropdownItem[];
+  position?: 'absolute' | 'static';
+  buttonIcon?: JSX.Element;
+};
+
+export const DropdownFilterMenu = ({
+  label,
+  items,
+  position = 'absolute',
+  buttonIcon: ButtonIcon,
+}: DropdownProps) => {
+  const [isActive, setIsActive] = useState(false);
+  const positionClass = `dropdown-${position}`;
+
+  const onToggle = () => {
+    setIsActive(!isActive);
+  };
+
+  return (
+    <div className="wrapper">
+      <Button variant="outline" size="md" color="tertiary" onClick={onToggle}>
+        {ButtonIcon}
+        {label}
+      </Button>
+      {isActive && (
+        <div className={`dropdown ${positionClass}`}>
+          <ul>
+            {items.map(({ label, field, onChange }) => (
+              <li key={field}>
```

Comment:
There is nothing to make sure this key is unique, u can do `${label}-${field}-${index}`